js/anxpro.js

```diff
@@ -80,6 +122,332 @@ module.exports = class anxpro extends Exchange {
+        const status = this.parseTransactionStatus (transactionState);
+        return {
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'id': this.safeString (transaction, 'uuid'),
+            'currency': code,
+            'amount': amount,
+            'address': address,
+            'tag': tag,
+            'status': status,
+            'type': type,
+            'updated': updated,
+            'txid': this.safeString (transaction, 'coinTransactionId'),
+            'fee': {
+                'cost': this.safeFloat (transaction, 'fee'),
+                'currency': code,
+            },
+            'info': transaction,
+        };
+    }
+
+    parseTransactionStatus (status) {
+        const statuses = {
+            'PROCESSED': 'complete',
+            'REVERSED': 'canceled',
+            'CANCELLED_INSUFFICIENT_FUNDS': 'canceled',
+            'CANCELLED_LIMIT_BREACH': 'canceled',
+        };
+        return this.safeString (statuses, status, 'pending');
+    }
+
+    async fetchMyTrades_v3 (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        //    { trades:
+        //     [ { tradeId: 'c2ed821d-717a-4b7e-beb0-a9ba60e8f5a0',
+        //         orderId: '5a65ae21-c7a8-4009-b3af-306c2ad21a02',
+        //         timestamp: '1551357057000',
+        //         tradedCurrencyFillAmount: '0.09006364',
+        //         settlementCurrencyFillAmount: '270.96',
+        //         settlementCurrencyFillAmountUnrounded: '270.96000000',
+        //         price: '3008.53930',
+        //         ccyPair: 'BTCEUR' },
+        //         { tradeId: 'fc0d3a9d-8b0b-4dff-b2e9-edd160785210',
+        //             orderId: '8161ae6e-251a-4eed-a56f-d3d6555730c1',
+        //             timestamp: '1551357033000',
+        //             tradedCurrencyFillAmount: '0.06521746',
+        //             settlementCurrencyFillAmount: '224.09',
+        //             settlementCurrencyFillAmountUnrounded: '224.09000000',
+        //             price: '3436.04305',
+        //             ccyPair: 'BTCUSD' },
+        //         { tradeId: '293f0893-111b-4c64-b036-9403d5c7422a',
+        //             orderId: '9d5e0850-db5f-4871-9c3b-3701a33315e4',
+        //             timestamp: '1551357005000',
+        //             tradedCurrencyFillAmount: '0.06277435',
+        //             settlementCurrencyFillAmount: '161.95',
+        //             settlementCurrencyFillAmountUnrounded: '161.95000000',
+        //             price: '2579.87538',
+        //             ccyPair: 'BTCGBP' } ],
+        //         count: 3,
+        //     timestamp: '1557438456732',
+        //     resultCode: 'OK' }
+        const request = {};
+        if (limit !== undefined)
+            request['max'] = limit;
+        const response = await this.v3privatePostTradeList (this.extend (request, params));
+        if (response['resultCode'] !== 'OK')
```

Comment:
We will need to move the checks for errors to the `handleErrors` below. It may need an overhaul as well, see how it is usually done with the exceptions mapping here:

- https://github.com/ccxt/ccxt/blob/master/js/bitfinex.js#L309
- https://github.com/ccxt/ccxt/blob/master/js/bitfinex.js#L1072

Would be great if you could help rewrite `handleErrors` to generalize the error handling for anxpro v1 / v3

Reply:
Can we do this later?  It's not really a part of my PR which was to add functionality

Reply:
@npomfret sure, I'll add the handler soon! I'd suggest to move on to the next one, and I'll finish this one and will merge shortly (I hope).

Reply:
@npomfret thx for your help on this, really appreciate the effort!